pages/checkoutPage.tsx

```diff
@@ -98,7 +106,7 @@ const CheckoutPage = () => {
               title={item.descriptor.name}
               description={item.descriptor.short_desc}
               quantity={item.quantity}
-              price={`${locale === "en" ? "Rs." : "£"} ${item.totalPrice}`}
+              price={`${locale === "en" ? "Rs." : "£"}${item.totalPrice}`}
```

Comment:
can we store this locale mapping on config/constant file.

Reply:
it is just a test implementation. We will be doing global change for this when we will pick translation task